INSTALL

```diff
@@ -1,288 +1,368 @@
-NMZ_MAC_STATIC: if set a Mac binary is built with dependency on libomp.dylib relative to the binary
-NO_OPENMP: if set blocks OpenMP in the compilation of Normaliz (and e-antic)
+'--version'
+'-V'
+     Print the version of Autoconf used to generate the 'configure'
+     script, and exit.
 
-
-GMP_INSTALLDIR: sets the path to the GMP files
-
-NMZ_OPT_DIR: sets the directory to which the sources of the optional libraries are downloaded;
-  by default it is the subdirectory "nmz_opt_lib"
-
-NMZ_MAC_STATIC: if set, a Mac binary is built with dependency on libomp.dylib relative to the binary
-
-EXTRA_FLAGS: One can further control the build process by setting variables of configure using this
-   variable, the contents of which are appended to the call to the configure script; see
-
-   ./configure --help
-
-Have a look at the install scripts.
-
-*****************************
-****   TROUBLESHOOTING   ****
-*****************************
-
-In order to produce the semistatic binary (Linux and MacOS) install_normaliz.sh must make
-the dynamic libraries invisble: after copying them to the subdirectory save_dynamic, 
-removes them /local/lib and restores them after the build.
-
-If you should encounter any problems at this point, manual intervention is required.
-Since the dynamic libraries have been copied to /local/lib/save_dynamic before
-deletion, you can copy them back to /local/lib. If the worst comes to the worst, you
-must re-install the optional packages.
-
-*********************
-****     SCIP    ****
-*********************
-
-To build SCIP:
-
-* download the scipoptsuite at http://scip.zib.de/ . 
-
-   Notice that SCIP
-   is not distributed under GPL, but the ZIB Academic License
-   (http://scip.zib.de/academic.txt).
-   
-* Unpack it and then compile it with
- 
-   make ZLIB=false GMP=false READLINE=false scipoptlib
-   
-   (To compiling scip for 32bit systems, add also ARCH=x86 to the make.)
-   
-   If your C++ compiler is too old, you may need to add
-   CXXFLAGS="-std=c++0x -fPIC"
-   
-* IMPORTANT: ** Don't try to build scipopt as a shared library!  It does not work
-      because their build system is broken **
-      
-For configure you must set
-      
-   --with-scipoptsuite-src=DIR
-   
-to name the directory where SCIP can be found.
-
-
-********************
-**** plain MAKE ****
-********************
-
-Under Linux one can still use our old simple plain Makefile.
-
-IT IS MEANT FOR THE DEVELOPMENT OF NORMALIZ and the creation of static binaries under Linux.
-
-This Makefile is mainly used for development and debugging. It builds statically linked binaries
-with -g (can be changed in Makefile.configuration).
-
-It is assumed that the optional libraries are installed in
-
-   local
-   
-within the Normaliz directory.
-
-1) Go to the source directory
-
-2) Check the configuration in Makefile.configuration, especially the locations of
-the optional packages and their versions.
-
-3) Say 
-
-        make -f Makefile.classic
```

Comment:
What is the benefit of this Makefile over the automake Makefile? @w-bruns

Reply:
I can't make a fully static Linux binary via autotools. For this purpose I need Makefile.classic.
But perhaps you know a way to produce fully static binaries via autotools.

Reply:
This seems to work for me:
```
❯ LDFLAGS="--static" ./configure --disable-shared --enable-static
❯ make
❯ ldd ./source/normaliz
	not a dynamic executable
```